aries-site/src/examples/templates/list-views/ListOrderExample.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+import { Box, Button, List, Menu, Text } from 'grommet';
+
+const movies = [
+  { title: 'Ponyo', genre: 'Fantasy/Adventure' },
+  { title: 'Spritied Away', genre: 'Family/Fantasy' },
+  { title: 'Grave of the Fireflies', genre: 'Fantasy/Adventure' },
+  { title: "Wayne's World", genre: 'Comedy/Music' },
+  { title: 'Detroit Rock City', genre: 'Comedy/Music' },
+  { title: 'Home Alone 2', genre: 'Comedy/Family' },
+  { title: 'The Lion King', genre: 'Family/Musical' },
+  { title: 'Mulan', genre: 'Action/Adventure' },
+  { title: 'Inside Out', genre: 'Comedy/Family' },
+];
+
+export const ListOrderExample = () => {
+  const [ordered, setOrder] = useState(movies);
+  const [allowReorder, setAllowReorder] = useState(false);
+
+  return (
+    <Box align="center" pad="large" role="application">
```

Comment:
What does `role="application"` do? is this necessary for users to have in their application or is it just specific to the DS site context. If it's the latter, I'd be inclined to remove

Reply:
`role="application"` makes a screen reader not take over arrow keys. This is necessary for any webapp that is trying to be more of an application. In this case it allows lists to get arrow key navigation when the screen reader is active so they can move through the up/down arrow controls (or active list items in the case of selectable list items.)